lib/plug/static.ex

```diff
@@ -112,43 +112,43 @@ defmodule Plug.Static do
   end
 
   def init(opts) do
-    at     = Keyword.fetch!(opts, :at)
-    from   = Keyword.fetch!(opts, :from)
-    gzip   = Keyword.get(opts, :gzip, false)
-    brotli = Keyword.get(opts, :brotli, false)
-    only   = Keyword.get(opts, :only, [])
+    at = Keyword.fetch!(opts, :at)
+    from = Keyword.fetch!(opts, :from)
+    gzip? = Keyword.get(opts, :gzip, false)
+    brotli? = Keyword.get(opts, :brotli, false)
+    only = Keyword.get(opts, :only, [])
     prefix = Keyword.get(opts, :only_matching, [])
```

Comment:
I have no particular opinion on it but I think we should make a global decision for all projects. Otherwise the situation will be worse where some maintainers add spaces and others remove them :).